package.json

```diff
@@ -31,10 +33,12 @@
     "express": "4.17.1",
     "express-session": "1.17.1",
     "flash": "1.1.0",
+    "forever": "^3.0.4",
```

Comment:
Make sure to lock the version numbers. I've had trouble in the past with auto-versioning for people trying out the demos. You can add `save-exact=true` to your `~/.npmrc` file to always just have it save the exact version numbers.